uint64 timestamp    # time since system start (microseconds)
float32 linear_velocity
float32 angular_velocity
uint8 vehicle_state
uint8 control_mode
uint16 error_code
float32 battery_voltage
bool light_control_enable
uint8 front_light_mode
uint8 front_light_custom_value
uint8 rear_light_mode
uint8 rear_light_custom_value
